pkg/core/mempool/mem_pool.go

```diff
@@ -287,6 +300,7 @@ func (mp *Pool) RemoveStale(isOK func(*transaction.Transaction) bool) {
 	sort.Slice(newClaims, func(i, j int) bool {
 		return newClaims[i].Cmp(newClaims[j]) < 0
 	})
+	mp.staleTxs = staleTxs
```

Comment:
And here you'll just do
```
go resendTransactions(staleTxs)
```
That will send them into the goroutine provided earlier.

Reply:
Fixed.